ludwig/features/base_feature.py

```diff
@@ -82,9 +82,11 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def create_input(self):
-        return tf.keras.Input(shape=self.get_input_shape(),
-                              dtype=self.get_input_dtype(),
-                              name=self.name + '_input')
+        # return tf.keras.Input(shape=self.get_input_shape(),
+        #                      dtype=self.get_input_dtype(),
+        #                      name=self.name + '_input')
+        return torch.Tensor(shape=self.get_input_shape(),
```

Comment:
We probably don't need to modify this stuff for now. This isn't needed for PyTorch.

Reply:
put the old code back